vscode-react/src/components/aKDeploymentTableBody.component.tsx

```diff
@@ -188,13 +198,10 @@ export const AKDeploymentTableBody = ({ deployments }: { deployments?: Deploymen
 						></div>
 					)}
 					<div
-						className="relative codicon codicon-trash cursor-pointer ml-2 z-20"
-						onClick={(e) => {
-							const refElement = e.currentTarget;
-							showPopper("deploymentDelete");
-							deletePopperElementRef.current = refElement;
-							setDeleteDeploymentId(deployment.deploymentId);
-						}}
+						className={`relative codicon codicon-trash ${
+							deployment.state === DeploymentState.ACTIVE_DEPLOYMENT ? "cursor-not-allowed" : "cursor-pointer"
+						} ml-2 z-20`}
+						onClick={(event) => deleteDeploymentDisplayPopper(event, deployment)}
```

Comment:
you mean show delete deployment popper ? 
or you mean you delete the popper ?

Reply:
renamed